src/pal/tests/palsuite/c_runtime/__iscsym/test1/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ add_executable(paltest_iscsym_test1
 add_dependencies(paltest_iscsym_test1 coreclrpal)
 
 target_link_libraries(paltest_iscsym_test1
-  pthread
```

Comment:
I can make that change, but I think it is not necessary because [according to the documentation](https://cmake.org/cmake/help/v3.0/command/target_link_libraries.html) `target_link_libraries` is transitive.

All these tests depend on `coreclrpal` and `corepal` has a link dependency on `pthread`, so it should work. (I guess the CI build still passing is kind of a confirmation of that theory)

That said, I like the idea of `COMMON_TEST_LIBRARIES` because it removes a lot of common declarations. Let me see if I can make that work.